src/container/container_iv.c

```diff
@@ -1008,7 +1008,7 @@ cont_iv_hdl_fetch(uuid_t cont_hdl_uuid, uuid_t pool_uuid,
 	arg.eventual = eventual;
 	arg.invalidate_current = invalidate_current;
 	rc = dss_ult_create(cont_iv_capa_refresh_ult, &arg, DSS_XS_SYS,
-			    0, 0, NULL);
+			    0, DSS_DEEP_STACK_SZ, NULL);
```

Comment:
I think we'd investigate the IV operations deeper, and try to fix all the potential stack overrun issues together:

1. Do we know what are the major stack consumers for the IV operations? Are these functions from Cart or DAOS? If they are from Cart, the async IV operation in lazy mode should use deep stack size as well? See iv_op_async().
2. Should we use deep stack size for all container IV operations? See ds_cont_tgt_snapshots_refresh().
3. Should we use deep stack size for pool IV operations as well? See ds_pool_iv_svc_fetch().

Reply:
I'm fine with landing it as-is if it's blocking 2.4.2. Let's address my concerns in master.